unified_planning/test/test_remove_quantifier_in_traj_constraint.py

```diff
@@ -0,0 +1,126 @@
+# Copyright 2021 AIPlan4EU project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import unified_planning
+from unified_planning.engines.compilers.trajectory_constraints_remover import (
+    TrajectoryConstraintsRemover,
+)
+from unified_planning.model.walkers import Simplifier, ExpressionQuantifiersRemover
+from unified_planning.shortcuts import *
+from unified_planning.test import TestCase
+
+
+class TestRemoveQuantifierInTrajConstraint(TestCase):
+    def setUp(self):
+        TestCase.setUp(self)
+        problem = self.define_problem()
+        self.simplifier = Simplifier(problem.env)
+        self.problem = problem
+        self.traj_remover = TrajectoryConstraintsRemover()
+        self.traj_remover._expression_quantifier_remover = ExpressionQuantifiersRemover(
+            problem.env
+        )
+        self.traj_remover._problem = self.problem
```

Comment:
This is not the correct way to use a `Compiler`, use the `compile` method instead